tests/test_buffer_property.py

```diff
@@ -0,0 +1,41 @@
+"Contains a collection of pytest tests that validates the buffer property."
+import nidaqmx
+from nidaqmx.constants import SampleTimingType
+from nidaqmx.errors import DaqError
+
+
+def test__buffer__set_int32_property__value_is_set(any_x_series_device):
+    """Test for validating int32 attributes in buffer."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+        task.timing.samp_timing_type = SampleTimingType.SAMPLE_CLOCK
+
+        # Setting a valid input buffer size of type int32
+        task.in_stream.input_buf_size = 2000000000
+        assert task.in_stream.input_buf_size == 2000000000
+
+
+def test__buffer__set_invalid_int32_value__default_value_is_retained(any_x_series_device):
+    """Test for validating int32 attributes in buffer."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+        task.timing.samp_timing_type = SampleTimingType.SAMPLE_CLOCK
+
+        # Setting a invalid input buffer size greater than int32
+        try:
+            task.in_stream.input_buf_size = 4000000000
+        except DaqError:
+            assert task.in_stream.input_buf_size == 2000000000
```

Comment:
This passes if no exception is raised. Use [pytest.raises](https://docs.pytest.org/en/4.6.x/reference.html#pytest-raises)